plugins/modules/site_intent.py

```diff
@@ -934,75 +941,78 @@ def get_diff_deleted(self, config):
             self.delete_single_site(item['id'], item['name'])
 
         # Delete the final parent site
-        self.delete_single_site(site_id, self.want.get("site_name"))
-        msg = "Site - {0} and it's child sites deleted successfully".format(self.want.get("site_name"))
-        self.result['response'] = msg
-        self.log(msg)
+        self.delete_single_site(site_id, site_name)
+        self.msg = "Site - {0} and it's child sites deleted successfully".format(site_name)
+        self.result['response'] = self.msg
+        self.log(self.msg, "INFO")
 
         return self
 
     def verify_diff_merged(self, config):
         """
-        Verify the merged status(Creation/Updation) of site configuration in Cisco DNA Center.
+        Verify the merged status(Creation/Updation) of site configuration in Cisco Catalyst Center.
         Args:
-            - self (object): An instance of a class used for interacting with Cisco DNA Center.
+            - self (object): An instance of a class used for interacting with Cisco Catalyst Center.
             - config (dict): The configuration details to be verified.
         Return:
-            - self (object): An instance of a class used for interacting with Cisco DNA Center.
+            - self (object): An instance of a class used for interacting with Cisco Catalyst Center.
         Description:
-            This method checks the merged status of a configuration in Cisco DNA Center by retrieving the current state
+            This method checks the merged status of a configuration in Cisco Catalyst Center by retrieving the current state
             (have) and desired state (want) of the configuration, logs the states, and validates whether the specified
-            site exists in the DNA Center configuration.
+            site exists in the Catalyst Center configuration.
         """
 
         self.get_have(config)
-        self.log(str(self.have))
-        self.log(str(self.want))
+        self.log("Current State (have): {0}".format(str(self.have)), "INFO")
+        self.log("Desired State (want): {0}".format(str(self.want)), "INFO")
+
         # Code to validate dnac config for merged state
         site_exist = self.have.get("site_exists")
+        site_name = self.want.get("site_name")
 
         if site_exist:
             self.status = "success"
-            msg = "Requested Site - {0} present in Cisco DNA Center and creation verified.".format(self.want.get("site_name"))
-            self.log(msg)
+            self.msg = "Requested Site - {0} present in Cisco Catalyst Center and creation verified.".format(site_name)
```

Comment:
self.message = "The requested site '{0}' is present in the Cisco Catalyst Center and its creation has been verified.".format(site_name)

Reply:
added.